BibBuddy/app/src/main/java/de/bibbuddy/BookFragment.java

```diff
@@ -46,7 +55,23 @@
 
   private BookDao bookDao;
   private NoteDao noteDao;
+
   private ExportBibTex exportBibTex;
+  private ImportBibTex importBibTex;
+
+  private String[] storageManifestPermissions = {
```

Comment:
Maybe you can make the String Array final because it won't change anymore.
```suggestion
  private final String[] storageManifestPermissions = {
```

Reply:
I will not change it, because e.g. in SearchModel we used an array that requires looking at the values. I will add comments to these lines, as in the SearchModel class.